loops.js

```diff
@@ -88,4 +102,19 @@
 // ==========================================
 
 
+for (let i = 1; i < 100; i++) {
+    switch (true) {
+        case i % 3 === 0 && i % 5 === 0:
+            console.log("FizzBuzz")
+            break;
+        case i % 3 === 0:
+            console.log("Fizz")
+            break;
+        case i % 5 === 0:
+            console.log("Buzz")
+            break;
+        default:
+            console.log(i)
+    }
+}
 
```

Comment:
prima uitgewerkt! ik ben zelf geen voorstander van switch, dit had ook in een if/else gekund.